src/components/PasteFailModalElement.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+export interface PasteFailModalElementProps {
+  message: string;
+  removed?: string[];
+}
+
+export class PasteFailModalElement extends React.PureComponent<PasteFailModalElementProps, {}> {
+
+  render() {
+    const { message, removed } = this.props;
+
+    return (
+      <div>
+        <div>
+          <p>{message}</p>
+        </div>
+        <ul>
+          {removed.map(r => (
```

Comment:
"removed" is declared to be an optional property.  If it is ever omitted, calling "map" here will generate a run-time error.   I think it is probably safe to just make removed not be optional.

Reply:
Good catch. Fixed.